packages/application-components/src/utils/is-uuid.ts

```diff
@@ -0,0 +1,6 @@
+const isUUID = (string: string): boolean =>
```

Comment:
Maybe we should use the `uuid.validate` function

https://www.npmjs.com/package/uuid#uuidvalidatestr